pkg/server/container_create.go

```diff
@@ -818,8 +814,8 @@ func generateApparmorSpecOpts(apparmorProf string, privileged, apparmorEnabled b
 }
 
 // Ensure mount point on which path is mounted, is shared.
-func ensureShared(path string, mountInfos []*mount.Info) error {
-	sourceMount, optionalOpts, err := getSourceMount(path, mountInfos)
+func ensureShared(path string, lookupMount func(string) (mount.Info, error)) error {
+	sourceMount, optionalOpts, err := getSourceMount(path, lookupMount)
```

Comment:
Do we still need `getSourceMount`? I think we could just call `lookupMount` directly here.

Reply:
Done.